### ISSUE TITLE
User story - Job creation

### ISSUE THREAD

**Issue by stephendawsondev** [stephendawsondev/StackPortfolio#29]
### If applicable, add a related Epic.

Epic: #28 

### User Story

As a **recruiter user**, I can **create a job post** so that I can **to attract the attention of potential applicants**.

### Acceptance Criteria

- The Recruiter has can easily access the form to post a job
- All the job post fields are relevant

### Please ensure you've completed these tasks before marking the issue as done.

- [x] Reviewed the user story for clarity and completeness.
- [x] Identified and linked related issues or epics.
- [x] Created or updated the necessary documentation.
- [ ] Written unit and/or integration tests for functionality.
- [ ] All code changes have been manually tested to ensure they meet the acceptance criteria.
- [x] Updated the issue with any relevant notes or feedback (to date).
- [x] Merged into the main branch and deployed to production (if applicable).

### Please add any further tasks that need to be completed before the issue can be marked as done.

- [x] Create a JobPost model
- [x] Add the url path for `/job-post/new` or similar
- [x] Create the create job post form template and UI

**Comment by stephendawsondev** [comment]
This has been completed in PR #85 - tests will be added later.
